XenModel/XenAPI-Extensions/Host.cs

```diff
@@ -521,6 +521,16 @@ public static bool RestrictSslLegacySwitch(Host h)
             return h._RestrictSslLegacySwitch;
         }
 
+        private bool _RestrictLivePatching
+        {
+            get { return BoolKey(license_params, "restrict_live_patching"); }

```

Comment:
We need to call BoolKeyPreferTrue() here, so that the deature is consideted restricted if the license key is not present.